index.js

```diff
@@ -594,8 +595,8 @@ export default class VideoPlayer extends Component {
 
 VideoPlayer.propTypes = {
   video: Video.propTypes.source,
-  thumbnail: Image.propTypes.source,
-  endThumbnail: Image.propTypes.source,
+  thumbnail: ImagePropTypes.source,
```

Comment:
Could we use `ComponentProps<typeof Image>['source']`? That way we don't need the deprecated types at all.

Reply:
Could you explain what you mean by "source"?

Reply:
Could you explain what you mean by "source"?

Reply:
Ohw, I'm sorry. This isn't TypeScript of course.